api/kessel/inventory/v1beta1/rhel_hosts_service.proto

```diff
@@ -13,38 +12,28 @@ option java_package = "org.project_kessel.api.inventory.v1beta1";
 
 message CreateRhelHostRequest {
   // The Rhel Host to create in Kessel Asset Inventory
-  RhelHost host = 1 [
-    (google.api.field_behavior) = REQUIRED
-  ];
+  RhelHost host = 1;
 }
 
 message CreateRhelHostResponse {
   // The Rhel Host created in Kessel Asset Inventory
-  RhelHost host = 1 [
-    (google.api.field_behavior) = REQUIRED
-  ];
+  RhelHost host = 1;
```

Comment:
IIUC this host shouldn't be here as we don't return anything in the response. Is that right?
```suggestion
```

Reply:
I did not change the hosted resources return as part of this PR.  There are some other changes that need to happen for that to be removed

Reply:
Sounds good, I can take care of it in a follow up